internal/filesystem/filesystem.go

```diff
@@ -48,6 +53,31 @@ func WriteFile(path string, data []byte) error {
 	return nil
 }
 
+func ReadFile(path string) ([]byte, error) {
+	fs := FileSystem()
+	return afero.ReadFile(fs, path)
+}
+
+func GetFromYaml(key string) (string, error) {
```

Comment:
I don't think this should be necessary. You should be able to use Viper to load and unmarsahl the config. There should be an example in the config.go file.